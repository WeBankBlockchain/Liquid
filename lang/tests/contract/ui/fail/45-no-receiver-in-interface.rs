use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    extern "liquid" {
        fn bar() -> u8;
    }
}

fn main() {}
