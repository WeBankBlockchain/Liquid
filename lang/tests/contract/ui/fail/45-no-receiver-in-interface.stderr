error: `&self` or `&mut self` is mandatory first parameter for all contract methods
 --> $DIR/45-no-receiver-in-interface.rs:6:9
  |
6 |         fn bar() -> u8;
  |         ^^^^^^^^^^^^^^
